### DISCUSSION TITLE
History Widget, (not Calendar)

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#1085] [Feature Requests]
The widgets for Loop Habit Tracker include Habit Streaks, Strength, Frequency, and "History". What's called History as a widget is called Calendar in the app. Can you please add the bar chart as a widget - the chart you call History in the app? That would be much appreciated.

![Screenshot_20210830-091246_Habits](https://user-images.githubusercontent.com/34518046/131318956-acf969f4-6aef-4fe1-9e83-0595b7f93eb9.png)

**Comment by hiqua** [comment]
Makes sense, we should:
* rename the "history" widget to "calendar" (or the "calendar" section in the app to "history")
* create the widget equivalent of the currently "calendar" part of the app

**Reply by ghost** [reply]
Thank you the reply. That would be fantastic, thanks!

**Comment by haszeldine** [comment]
It would be great to have that widget; it's especially useful for numeric habits. Are there any current plans to implement this request?
